### DISCUSSION TITLE
Local dimming

### DISCUSSION THREAD

**Discussion by thehhmdb** [alvr-org/ALVR#1500] [General]
Would it be possible to have an option to enable/disable local dimming on the Quest Pro with the XR_META_local_dimming extension?

I believe the ALXR client version always has this toggled on.

**Comment by zmerp** [comment]
Not yet unfortunately. ALVR is using [openxrs](https://github.com/Ralith/openxrs) and it supports only published OpenXR extensions. `XR_META_local_dimming` (and also those for eye tracking and face tracking) are not yet published as part of an official OpenXR version.

**Comment by thehhmdb** [comment]
Looking at OpenXR specification release notes, I believe `XR_META_local_dimming` is in the specification now

**Reply by zmerp** [reply]
I did not miss this, but I didn't have the time to implement it yet. But you should expect support soon.
